docs/working-with-node/query-json-rpc.md

```diff
@@ -29,6 +29,10 @@ polygon-edge genesis --premine 0x1010101010101010101010101010101010101010:0x1231
 
 The balance can be either a `hex` or `uint256` value.
 
+:::warning Only premine accounts from which you have a private key!
```

Comment:
Typo here, should be `for which....`

Reply:
Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870